### PR TITLE
NO-ISSUE - add HostStageStartWaitingForControlPlane to dissconnection…

### PR DESCRIPTION
**Pull request by oshercc**
… validation

### CODE REVIEW

**Comment by oshercc** [comment]
/hold
waiting for ci stabilization

File: internal/host/statemachine.go [hunk 1]
```diff
@@ -310,7 +310,7 @@ func NewHostStateMachine(th *transitionHandler) stateswitch.StateMachine {
 	sm.AddTransition(stateswitch.TransitionRule{
 		TransitionType: TransitionTypeRefresh,
 		SourceStates: []stateswitch.State{
-			stateswitch.State(models.HostStatusInstalling),
```
Comment (filanov) [inline internal/host/statemachine.go:313]: We need both.

**Comment by filanov** [comment]
Please add a test for preparing for installation.

File: internal/host/statemachine.go (hunk 1 above)
Comment (filanov) [reply internal/host/statemachine.go:313]: still needed.

File: internal/host/transition_test.go [hunk 2]
```diff
@@ -1293,12 +1294,46 @@ var _ = Describe("Refresh Host", func() {
 				Expect(swag.StringValue(resultHost.Status)).To(Equal(models.HostStatusError))
 				info := statusInfoConnectionTimedOut
 				Expect(swag.StringValue(resultHost.StatusInfo)).To(MatchRegexp(info))
-
 			})
-
 		}
 	})
 
+	Context("host disconnected & preparing for installation", func() {
```
Comment (filanov) [inline internal/host/transition_test.go:1301]: context is not really needed for a single test case.

**Review by filanov — APPROVED** [review]

**Pull request merged**